data/changelog.txt

```diff
@@ -51,6 +51,12 @@ Sound-triggered traps
 Item transformation can now pick variant items to transform into
 You can now collect grappling hooks and ladders from the ledge above.
 Adds priority parameter for special placement
+Professions can start with multiple martial arts
+Microlab Mapgen is now parameter based.
+Allow NPC_DEATH EoC to prevent npc die
+Allow NPC doctors to install and remove CBMs from player allies
+Add 3 variants of roadblock and rebalanced loot
+Look up and down with 3d vision off
```

Comment:
```suggestion
Professions can start with multiple martial arts
Microlab Mapgen is now parameter based.
Allow NPC_DEATH EoC to prevent npc die
Allow NPC doctors to install and remove CBMs from player allies
Look up and down with 3d vision off
```